shopping-cart/deploy/scripts/common/builds.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 
+if [[ "$(basename "${0#-}")" = "$(basename "${BASH_SOURCE[0]}")" ]]; then
+  echo "This is a bash source library, not a bash script!" >&2
+  exit 1
+fi
+
 setTag() {
-    ## Assuming `docker images` sorts in reverse chronological order (and probably it'll be pointing to an 
-    ## empty repository anyway) we list all images, filter by non-latest and filter by name. Finally, take 
+    ## Assuming `docker images` sorts in reverse chronological order (and probably it'll be pointing to an
+    ## empty repository anyway) we list all images, filter by non-latest and filter by name. Finally, take
     ## only the first match (which is the recently built).
-    export IMAGE_TAG=`docker images  |grep shopping-cart|grep -v latest | awk -F\  '{print $2}'| head -1`
+    IMAGE_TAG=`docker images  |grep shopping-cart|grep -v latest | awk -F\  '{print $2}'| head -1`
```

Comment:
FWIW, one of the failure modes I saw was that the image fails to build, then this command returned an empty string, `IMAGE_TAG` was set to an empty string, then later in the build, it fails with the rather cryptic error: "Error parsing reference: "shopping-cart:" is not a valid repository/tag: invalid reference format" .

https://travis-ci.com/lagom/lagom-samples/jobs/246588577#L883-L886